db/knexfile.js

```diff
@@ -0,0 +1,53 @@
+// Update with your config settings.
+
+module.exports = {
+
+  development: {
+    client: 'mysql',
+    connection: {
+      database: 'gallery',
+      user:     'root',
+      password: ''
+    },
+    pool: {
+      min: 2,
+      max: 10
+    },
+    migrations: {
+      tableName: 'knex_migrations'
+    }
+  }
+
+  // staging: {
```

Comment:
Is this commented-out code going to be used at all or can it be deleted?

Reply:
Might be updated and used later when we have production vs development environments.